doc/commands.doc

```diff
@@ -1452,34 +1452,58 @@ contains \c TEST, or \c DEV
   | \%A | The full weekday name.
   | \%lA | As \%A but when not by default  the case and when possible the first character is converted to uppercase.
   | &nbsp; | &nbsp;
+  | \%H |The hour (0-23).
+  | \%0H | The hour (0-23). If the result is a single digit, it is prefixed with 0.
+  | \%I | The hour (0-11).
+  | \%0I | The hour (0-11). If the result is a single digit, it is prefixed with 0.
+  | \%M | The minute.
+  | \%0M | The minute. If the result is a single digit, it is prefixed with 0.
+  | \%S | The second.
+  | \%0S | The second. If the result is a single digit, it is prefixed with 0.
+  | &nbsp; | &nbsp;
   | \%\% | A \% character.
 
   Note that the \<format\> has to be between double quotes.
 
-  In case the \<date\> is specified it to has to have the following representation:
-  - one or 2 digits for the day
-  - a minus sign
-  - one or 2 digits for the month
-  - a minus sign
-  - 4 digits for the year
+  In case the \<date_time\> is specified it has to have the following representation:
+  - optional `date` where `date` is:
+    - one or 2 digits for the day
+    - a minus sign
+    - one or 2 digits for the month
+    - a minus sign
+    - 4 digits for the year
+  - optional `time` where `time` is:
+    - whitespace
+    - one or 2 digits for the hours
+    - a colon sign
+    - one or 2 digits for the minutes
+    - optionally
+      - a colon sign
+      - 2 digits for the seconds
   .
-  in case the \<date\> is not specified the current date is used.
+  in case the \<date_time\> is not specified the current date and time are used.
 
   Here is an example:
   \code
   - \showdate "%lA %0d-%0m-%Y" 14-3-2015
   - \showdate "%a %d-%m-%y" 14-3-2015
   - \showdate "%m.%d%y" 14-3-2015
+  - \showdate "%lA %0d-%0m-%Y %H:%M:%S" 14-3-2015 03:04:15
+  - \showdate "%lA %0d-%0m-%Y %H:%M:%S" 14-3-2015 03:04
   \endcode
   In case `OUTPUT_LANGUAGE=english` this results in:
   - Saturday 14-03-2015
   - Sat 14-3-15
   - 3.1415
+  - Saturday 14-03-15 3:4:15
+  - Saturday 14-03-15 3:4:0
```

Comment:
why the 0 seconds? this is not according to the time format given.

Reply:
This is the default value used when not specified and the user specified `%H:%M:%S` as format so including seconds.

Reply:
ok, I see. 

Still a bit inconsistent as this seems to be the only value that when omitted in the format is replaced by a 0. Maybe it should just be an error, i.e. either omit the `:%S` or specify the seconds.

Reply:
It is the choice of the user in this case to have an output format the specifies to output the seconds but the user did not specify this value in its input.

Reply:
We can also conclude that what the user has specified is incomplete/wrong and give an error? 

The put it another way, if the user doesn't specify the minutes value should we not also fill in 0 there? Same for hour? i.e. why is the seconds value treated in a special way? Is there a use case for it?

Reply:
> We can also conclude that what the user has specified is incomplete/wrong and give an error?

could be concluded and can be implemented but will be a bit of work.

> The put it another way, if the user doesn't specify the minutes value should we not also fill in 0 there? Same for hour? i.e. why is the seconds value treated in a special way? Is there a use case for it?

It is more a gut feeling that the user is not interested in the seconds value / doesn't can / will specify the seconds but will specify the hour and minute.